app/api/v1/views/party.py

```diff
@@ -111,37 +118,42 @@ def patch_party(id, name):
         hqAddress = data.get('hqAddress')
         logoUrl = data.get('logoUrl')
 
-        if name == "":
+        if isBlank(name):
             return make_response(jsonify({
```

Comment:
This can be improved. Consider creating an error function that does the make_response and takes in parameters of the error message/

Reply:
Done!